gpu-alloc/src/block.rs

```diff
@@ -252,7 +252,7 @@ impl<M> MemoryBlock<M> {
 
             device.flush_memory_ranges(&[MappedMemoryRange {
                 memory: &self.memory,
-                offset: aligned_offset,
+                offset: self.offset + aligned_offset,
```

Comment:
actually, we need to fix this better. The `self.offset` should be aligned together with the offset, i.e.:
```rust
let aligned_offset = align_down(self.offset + offset, self.atom_mask);
```

Reply:
`self.offset` must already be aligned to `atom_mask`

Reply:
ok, so just land this fix then? Or are there other concerns?